src/normalizer.ts

```diff
@@ -222,6 +223,32 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable to null type', schema => {
+  if (schema.nullable !== true) {
+    return
+  }
+
+  delete schema.nullable
+
+  if (schema.const !== undefined) {
+    if (schema.const !== null) {
+      schema.enum = [schema.const, null]
+      delete schema.const
+    }
+  } else if (schema.enum) {
+    if (!schema.enum.includes(null)) {
+      schema.enum.push(null)
+      log('yellow', 'normalizer', 'enum should include null when schema is nullable', schema)
```

Comment:
@mdmower-csnw: 
1. Should there be a warning for the `const` case, too? [AJV](https://ajv.js.org/json-schema.html#nullable) docs say with `nullable: true`:
> const would fail, unless it is "const": null

2. I wouldn't be opposed to throwing instead of the warnings, because it's critical that the schema is updated or the type will be at odds with how AJV validates.

3. What if `const` and `enum` are both defined? Is that case caught earlier by validation?